content/docs/operating/storage.md

```diff
@@ -253,7 +295,9 @@ that cannot be recovered is moved to a directory called `orphaned`
 you do not need it anymore.
 
 The crash recovery usually takes less than a minute. Should it take much
-longer, consult the log to find out what has gone wrong.
+longer, consult the log to find out what is going on. With increasing number of
+time series in the storage (archived or not), the re-indexing tends to dominate
+the recovery time and can take tens of minutes.
```

Comment:
I'm going off my benchmarking, which was down to ~5m to do all the indexing stuff in my setup. It's certainly possible it's worse for crash recovery.